src/cljs/athens/electron.cljs

```diff
@@ -325,6 +325,52 @@
   ;; open or create a new starter db
 
   ;; Watch filesystem, e.g. in case db is updated via Dropbox sync
+  
+    ;; Update index.transit    
+  (defn write-db-index [file filepath]
+    (go   
+     (when-not (exists? (fs.writeFileSync filepath file))  
+     (js/alert "Backup file correctly restored, click Ok to re-start") 
```

Comment:
regarding styling: sub-forms should be indented by 2 spaces more than their parent, in cases such as this, I believe

Reply:
done